sbt-plugins/scalahost/src/main/scala/scala/meta/scalahost/sbt/ScalahostSbtPlugin.scala

```diff
@@ -0,0 +1,41 @@
+package scala.meta.scalahost.sbt
+
+import sbt._
+import Keys._
+import org.scalameta.BuildInfo
+import sbt.plugins.JvmPlugin
+
+object ScalahostSbtPlugin extends AutoPlugin {
+  override def requires = JvmPlugin
+  override def trigger: PluginTrigger = AllRequirements
+  // Defaults to builds version.value, but with possibility to override.
+  val scalahostVersion: String = sys.props.getOrElse("scalahost.version", BuildInfo.version)
+  // If set to output of scalahost/package, uses -Xplugin:/path/to/scalahost.jar
+  // instead of addCompilerPlugin. When testing scalahost locally, this provides a
+  // much faster iteration speed compared to publishLocal.
+  // NOTE. Only possible with one scalaVersion, not cross-build projects.
+  val scalahostPackageJar: Option[String] = sys.props.get("scalahost.packagejar")
+  override def projectSettings = Seq(
+    commands += Command.command("dumpsemanticdb")(s => "test:compile" :: s),
```

Comment:
Before this PR, I only ran test:compile for >2.11 scala version projects and printed out useful info. Since I don't do that anymore, I've remove this command.